react-ui/src/pages/submission/Submission.jsx

```diff
@@ -1,3 +1,9 @@
+/**
+ * User submission portal
+ * ??? no useEffect errors?
```

Comment:
What does this mean?

Reply:
Sorry, should have read something more clear like "Consider useEffect to manage error checking". Rather than waiting for the form to be submitted, you can check each field through useEffect as they are filled out.

Reply:
It's actually validated as it's being filled in the `handleFieldChanged` function, though perhaps that logic can perhaps be generalized into a `useEffect` hook.